pennylane/tape/qscript.py

```diff
@@ -1130,7 +1130,15 @@ def adjoint(self):
         adj = self.__class__(
             ops=ops_adj, measurements=self.measurements, prep=self._prep, shots=self.shots
         )
-        if self.do_queue:
+
+        if self.do_queue is not None:
+            do_queue_deprecation_warning = (
+                "The do_queue keyword argument is deprecated. "
+                "Use qml.queuing.QueuingManager.stop_recording()"
+            )
+            warnings.warn(do_queue_deprecation_warning, UserWarning)
```

Comment:
Reading this repeatedly gives me ideas of having a central `deprecations.py` file with e.g. warnings. This might make adding and removing deprecations more handy :) 
Just a side note, though :)

Reply:
Absolutely! In such a file we could also manage how often warnings are issued, e.g., using
```Python
warnings.filterwarnings("once", "The do_queue keyword argument is deprecated.", UserWarning)
```
In this PR I tried to make sure that the deprecation warning is not issued more than once, but sometimes that is impossible to achieve manually. @trbromley and I also talked about this.

Reply:
Ah, that's a nice feature of this as well :) 
Not time sensitive but definitely would be nice to have!

Reply:
(I'll resolve, because this topic is closed for this PR)